### ISSUE TITLE
Get train, validation, and test splits for STS patients who have all desired tensors

### ISSUE THREAD

**Issue by erikr** [aguirre-lab/ml4c3#1324]
## Question or modeling task
This will let us compare various models on exactly the same data.

**Comment by raimonpv** [comment]
I was thinking that we could implement such functionality in `explore` mode. When you set a specific parameter you filter the input `patient_csv` by the patients who have all the tensors.

**Comment by erikr** [comment]
This functionality exists; `explore` should produce two sets of outputs, `_intersect` and `_union`.

`_intersect` is for patients that have all tensors specified by the TensorMaps.

`_union` is for all patients.

**Comment by erikr** [comment]
@StevenSong @raimonpv We may want a cleaner way of generating `patient_csv`, `train`, `validation`, and `test` CSV files. Currently we have to:
1. Call `explore` using all desired TensorMaps.
2. Make a new script or notebook to parse `tensors_union.csv` and find rows that have all non-`NaN` values.
3. Save that list of MRNs to a CSV.
4. Use that CSV as the `patient_csv` in [`create-bootstraps.ipynb`](https://github.com/aguirre-lab/ml4c3/blob/master/notebooks/sts/create-bootstraps.ipynb)

**Comment by raimonpv** [comment]
When I said: 

> I was thinking that we could implement such functionality in explore mode. When you set a specific parameter you filter the input patient_csv by the patients who have all the tensors.

The output I was thinking was this one:
>  Make a new script or notebook to parse tensors_union.csv and find rows that have all non-NaN values.

And then filter the input `patient_csv` file with the resulting MRNs of the filtered `tensors_unions.csv` and save it. I think that this file should be generated always with `tensors_union.csv` so no additional argument would be required.

**Comment by erikr** [comment]
We used to save `tensors_intersect.csv` (in addition to `tensors_union.csv`), but it was redundant with `tensors_union.csv` and not used for anything, since we did not have the need described in this issue back when we first created `explore`.

I would be open to restoring that functionality. We don't use `explore` a lot, so it would not consume too much storage space even though the two `tensors_` files are redundant.
